### DISCUSSION TITLE
2023/08/my-process-for-reviewing-pull-requests/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#830] [Specific Post Comments]
# 2023/08/my-process-for-reviewing-pull-requests/

I often do a lot of code reviews and pull request reviews. As a follow-up to my article on submitting pull requests, I wanted to talk about how I review them as well.

https://seankilleen.com/2023/08/my-process-for-reviewing-pull-requests/

**Comment by RyKilleen** [comment]
"Look for things I like" is the single best PR tip I've received from you. It completely changed the tone of what can otherwise be a really vulnerable experience (especially for newer devs.)

Thanks for throwing these all in one place!
